seed@.js

```diff
@@ -0,0 +1,159 @@
+const restaurantsData = require('./fakedata10M.js');
+const reviewsList = require('./sample_reviews.js');
+const names = require('./names.js');
+const cities = require('./cities.js');
+const faker = require('faker');
+const fs = require('fs');
+const mongoose = require('mongoose');
+const db = require('./db/mongodb.js');
+
+const { exec } = require('child_process');
+
+
+let generateData = () => {
```

Comment:
Looks like there are two files in two different places here? Might want to get rid of one if they're identical

Reply:
yea I forgot to delete the old file once I got all the functionality working thanks bud